.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,81 @@
+name: Docker Image CD
+
+on:
+  workflow_dispatch:
+    inputs:
+      bin_tag:
+        description: "Okteto bin tag"
+        required: true
+        type: string
+
+jobs:
+  build_and_push:
+    name: Push to DockerHub
+    runs-on: "ubuntu-latest"
+
+    defaults:
+      run:
+        working-directory: .
+
+    steps:
+      - name: Checking out the repository
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Context for Buildx
+        id: buildx-context
+        run: |
+          docker context create builders
+
+      - name: Set up Docker Builder
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+          install: true
+          endpoint: builders
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@master
+        with:
+          platforms: arm64,amd64
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          username: "${{ secrets.DOCKER_USERNAME }}"
+          password: "${{ secrets.DOCKER_TOKEN }}"
+
+      - name: Login to GitHub Container Registry
```

Comment:
Let's just push to Dockerhub. One reason to use GH container registry is unlimited pulls, but we are verified publishers in Dockerhub and our images have unlimited pulls for everyone for free

Reply:
Perfect! I didn't know  about that feature in Dockerhub 👍🏻